src/view/components/QueryStateIndicators/WarningIconError.tsx

```diff
@@ -16,7 +16,7 @@ export const WarningIconError: FC<WarningIconErrorProps> = ({
         <WarningIcon fontSize={size} color="error" titleAccess="error" />
       </Grid>
       <Grid item>
-        <Typography variant={size === 'small' ? 'body1' : 'h6'} color="error">
+        <Typography variant={size === 'small' ? 'body1' : 'h6'} color="initial">
```

Comment:
`initial` is the [default value for the color prop](https://material-ui.com/api/typography/#props), so we don't need to specify that here.

```suggestion
        <Typography variant={size === 'small' ? 'body1' : 'h6'}>
```